Source/WebCore/inspector/agents/InspectorNetworkAgent.cpp

```diff
@@ -908,6 +908,10 @@ Protocol::ErrorStringOr<void> InspectorNetworkAgent::disable()
 
     setResourceCachingDisabled(false);
 
+#if ENABLE(INSPECTOR_NETWORK_THROTTLING)
+    setEmulatedConditions(0);
```

Comment:
Probably `std::nullopt` instead of `0`?

Reply:
Yeah i suppose we could just make it so that anyone calling `Network.setEmulatedConditions` has to provide values for everything they want to affect up front instead of having the backend maintain this sorta implicit state.